packages/commonwealth/client/scripts/views/components/component_kit/CWBannerText/CWBannerText.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { CWText } from '../cw_text';
+import './CWBannerText.scss';
+
+const urlRegex = /(https?:\/\/[^\s]+)/g;
+
+type TextProps = {
+  text: string | React.ReactNode;
+};
+
+const RenderTextWithLink = ({ text }: TextProps) => {
+  if (typeof text !== 'string') {
+    return <>{text}</>;
+  }
+  const parts = text.split(urlRegex);
+
+  return (
+    <>
+      {parts.map((part, index) => {
+        if (urlRegex.test(part)) {
+          return (
+            <React.Fragment key={index}>
+              <CWText type="b2" className="terms-text">
```

Comment:
terms-text here and below is not defined in the scss file so you shouldn't use it

Reply:
done this change